scenario.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, params, experiment_path, scenario_id=1, n_repeat=1, is_dry_ru
 
         # Raise Exception if unknown parameters in the .yml file
 
-        params_known = ["dataset_name","partners_count","amounts_per_partner","samples_split_option","multi_partner_learning_approach","aggregation_weighting","methods","gradient_updates_per_pass_count","is_quick_demo", "corrupted_datasets", "epoch_count", "minibatch_count", "is_early_stopping"]
+        params_known = ["dataset_name","partners_count","amounts_per_partner","samples_split_option","multi_partner_learning_approach","aggregation_weighting","methods","gradient_updates_per_pass_count","is_quick_demo", "corrupted_datasets", "epoch_count", "minibatch_count", "is_early_stopping", "proportion_dataset"]
```

Comment:
It would be easier to read and cleaner if you put it next to the closest one, in terms of concept. So I'd say right after `dataset_name`

Reply:
Agree with that! It could also be more readable to reformat this loooong list!

Reply:
I have reformatted the list in 5 parts